src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs

```diff
@@ -60,11 +70,24 @@ public string Format()
                 };
             }
 
+            JObject sessionHandlerOptions = null;
+            if (SessionHandlerOptions != null)
+            {
+                sessionHandlerOptions = new JObject
+                {
+                    { nameof(SessionHandlerOptions.AutoComplete), MessageHandlerOptions.AutoComplete },
```

Comment:
So when not set, you're defaulting to the MessageHandlerOptions values? Why not just accept the default SB SessionHandlerOptions? That might be what users would want/expect?

Reply:
It was copy/past mistake. Good catch.